src/composables/draw/draw.composable.ts

```diff
@@ -23,15 +24,16 @@ export default function useDraw() {
     drawPolygon: useDrawInteraction({ type: 'Polygon' }).drawInteraction,
   } as DrawInteractions
 
-  watch(drawStateActive, () => {
+  useDrawNotifications()
```

Comment:
unused ??
or do you have to call the object to enable reactivity ?

Reply:
This activates watchers inside `useDrawNotifications()` and with these watchers, the notification system.